packages/x-components/tests/e2e/cucumber/recommendations/recommendations.feature

```diff
@@ -1,8 +1,10 @@
 Feature: Recommendations component
 
+  Background:
+    Given a recommendations API with a known response
+
   Scenario Outline:  1. Recommendations are displayed
-    Given suggestions API should respond with mocked suggestions
-    And   following config: max items to store is <maxItemsToRequest>
+    Given following config: max items to store is <maxItemsToRequest>
     Then  number of rows requested in "<request>" is <maxItemsToRequest>
     Then  number of displayed recommendations are equal or less than <maxItemsToRequest>
```

Comment:
**And**  number of displayed recommendations are equal or less than <maxItemsToRequest>